src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -151,8 +191,8 @@ public void getExistingBrapiData(List<BrAPIImport> importRows, Program program)
                 List<String> pedigreeArray = List.of(germplasm.getPedigree().split("/"));
                 String femaleParent = pedigreeArray.get(0);
                 String maleParent = pedigreeArray.size() > 1 ? pedigreeArray.get(1) : null;
-                if (created.contains(femaleParent)){
-                    if (maleParent == null){
+                if (created.contains(femaleParent)) {
+                    if (maleParent == null) {
                         createList.add(germplasm);
                     } else if (created.contains(maleParent)) {
                         createList.add(germplasm);
```

Comment:
The outer if statement looks like it exists due to the fact that there is also the else if depending on it.

Reply:
Just to be sure we are talking about the thing since my previous comment wasn't very specific, my proposed change was to this:

```
if (created.contains(femaleParent)) {
  if (maleParent == null || created.contains(maleParent)) {
    createList.add(germplasm);
  }
}
```

Since both of inner if's contain the same action.

Reply:
Ah that makes sense. Implemented.